src/index.html

```diff
@@ -20,11 +20,109 @@
 	<link rel="icon" type="image/png" sizes="32x32" href="assets/icon/favicon-32x32.png">
 	<link rel="icon" type="image/png" sizes="96x96" href="assets/icon/favicon-96x96.png">
     <link rel="icon" type="image/png" sizes="16x16" href="assets/icon/favicon-16x16.png">
-    <link rel="manifest" href="assets/manifest.json">
+	<link rel="manifest" href="assets/manifest.json">
+	
+	<style>
+        body, html {
+            height: 100%;
+            margin: 0;
+        }
+        .cog {
+            left: 39px;
+            opacity: 0;
+            position: absolute;
+            top: 12px;
+        }
+        .loading-wrapper {
+            display: flex;
+            justify-content: center;
+            height: 100%;
+            align-items: center;
+            flex: 1;
+        }
+        .logo {
+            height: 55px;
+            position: relative;
+            width: 253px;
+        }
+        .spin {
+            -webkit-animation: spin 2s infinite linear, fade 2s normal forwards;
+            animation: spin 2s infinite linear, fade 2s normal forwards;
+        }
+        @-webkit-keyframes spin {
+            0% {
+                -webkit-transform: rotate(0deg);
+                transform: rotate(0deg);
+            }
+            100% {
+                -webkit-transform: rotate(359deg);
+                transform: rotate(359deg);
+            }
+        }
+        @keyframes spin {
+            0% {
+                -webkit-transform: rotate(0deg);
+                transform: rotate(0deg);
+            }
+            100% {
+                -webkit-transform: rotate(359deg);
+                transform: rotate(359deg);
+            }
+        }
+        @-webkit-keyframes fade {
+            0% {
+                opacity: 0;
+            }
+            100% {
+                opacity: 1;
+            }
+        }
+        @keyframes fade {
+            0% {
+                opacity: 0;
+            }
+            100% {
+                opacity: 1;
+            }
+        }
+    </style>
 </head>
 
 <body>
-    <app-root></app-root>
+	<app-root></app-root>
+	
+	<!-- Loading Indicator -->
+	<div class="loading-wrapper">
```

Comment:
You need to put the ```<div class="loading-wrapper">``` inside the ```<app-root>```

Like:

```html
<app-root>
    <div class="loading-wrapper">
        // WHATEVER
    </div>
</app-root>
```

If not, when the app loads, the html for the loading will be in the dom.